pillar/params.sls

```diff
@@ -23,6 +23,9 @@ dns:
 kube_user:        'kube'
 kube_group:       'kube'
 
+# Log level for kubernetes services, default is 2
+log_level:        '2'
```

Comment:
I agree with @kiall. We will have multiple components that would follow different conventions for configuring the log level. I would just use different variables for each of them: for example, `kube_log_level` for all the kubernetes daemons, `flannel:log_level` for flannel, and so on...

(at some moment we should move the kubernetes stuff to to something like `kube:log_level`)